e2e/autoscript-apps/src/authn-webauthn/autoscript.ts

```diff
@@ -36,6 +36,70 @@ function autoscript() {
   } catch (err) {
     // Do nothing
   }
+  /***
+   * Test Device Registration (Not Automated)
+   * must create cert because you cannot have a TLS error with webauthn to test manually
+   */
+  console.log('Click the device registration button!');
+  const deviceRegistration = document.querySelector('.device-registration');
+  fromEvent(deviceRegistration, 'click')
+    .pipe(
+      mergeMap(() => {
+        console.log('Initiate first step with `undefined`');
+        return forgerock.FRAuth.next();
+      }),
+      rxDelay(delay),
+      mergeMap((step) => {
+        console.log('Set username on auth tree callback');
+        console.log(step);
+        step.getCallbackOfType('NameCallback').setName(un);
+        return forgerock.FRAuth.next(step);
+      }),
+      rxDelay(delay),
+      mergeMap((step) => {
+        console.log('Set password on auth tree callback');
+        step.getCallbackOfType('PasswordCallback').setPassword(pw);
+        return forgerock.FRAuth.next(step);
+      }),
+      mergeMap(async (step) => {
+        const webAuthnStep = forgerock.FRWebAuthn.getWebAuthnStepType(step);
+        if (webAuthnStep === 2) {
+          console.log('WebAuthn step is registration');
+        } else {
+          throw new Error('WebAuthn step is incorrectly identified');
+        }
+        console.log('Handle WebAuthn Registration');
+        try {
+          step = await forgerock.FRWebAuthn.register<'mydevice'>(step, 'mydevice');
+          // ensure the step here has the 'mydevice' name at the end of the value. (outcome)
```

Comment:
No, this was instructions for when you get to this step, you can ensure the callback has the value with `mydevice` at the end since it concates with a stringifed object its messy.

see here:
<img width="1538" alt="Screenshot 2023-04-25 at 1 18 17 PM" src="https://user-images.githubusercontent.com/18267769/234380491-7268883f-0b58-40e7-9a74-5adaa5b9005b.png">

Reply:
Oh, okay. Rather than this comment instructing the manual tester to inspect the step in the console, can we just have a condition in the code to check this for us, similar to the other conditions that are ensuring things are working properly? If `mydevice` is not present, let's just throw an error like how we handle other issues.